### PR TITLE
add batch_size to Weaviate

### PR DESCRIPTION
**Pull request by baskaryan**
cc @mcantillon21 @hsm207 @cs0lar

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/weaviate.py [hunk 1]
```diff
@@ -447,9 +489,6 @@ def from_texts(
 
             batch.flush()
 
-        relevance_score_fn = kwargs.get("relevance_score_fn")
-        by_text: bool = kwargs.get("by_text", False)
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/weaviate.py:451]: @hsm207 this default to True in init and False in from_texts — is that on purpose?

Comment (hsm207) [reply libs/langchain/langchain/vectorstores/weaviate.py:451]: yes. it is on purpose. 

This issue has the context: https://github.com/langchain-ai/langchain/issues/4742

Comment (baskaryan) [reply libs/langchain/langchain/vectorstores/weaviate.py:451]: are you referring to this comment?

> can we have a way to toggle between the two? eg keep both functionality and let the user decide?

i get why the parameter exists, but why does it have different defaults depending on how you instantiate?

Comment (hsm207) [reply libs/langchain/langchain/vectorstores/weaviate.py:451]: I can't remember what the exact reason was anymore but not doing that will cause some examples in the demo notebook to break. 

Context: https://github.com/langchain-ai/langchain/pull/5201

Comment (baskaryan) [reply libs/langchain/langchain/vectorstores/weaviate.py:451]: we can always update notebook if needed — is there one that you'd strongly prefer to be default?

Comment (hsm207) [reply libs/langchain/langchain/vectorstores/weaviate.py:451]: I don't have a strong preference for a default value for `by_text`.

**Comment by angel-luis** [comment]
Hi, after this change I can't use `from_documents` with an existing index. It gives me the error:
```
weaviate.exceptions.UnexpectedStatusCodeException: Create class! Unexpected status code: 422, with response body: {'error': [{'message': 'class name "Test" already exists'}]}
```

A thing that has no sense since I'm loading more documents in an existent index.

**Pull request merged**